packages/node/src/indexer/indexer.manager.ts

```diff
@@ -173,8 +166,8 @@ export class IndexerManager extends BaseIndexerManager<
 
   private async indexReceipt(
     receipt: NearTransactionReceipt,
-    dataSources: SubqlProjectDs[],
-    getVM: (d: SubqlProjectDs) => Promise<IndexerSandbox>,
+    dataSources: NearProjectDs[],
+    getVM: (d: NearProjectDs) => Promise<IndexerSandbox>,
```

Comment:
line 184 `baseCustomHandlerFilter` not used?

Reply:
Its used by `BaseIndexerManager`